serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -18,52 +20,92 @@ class SerializabilityCheckerCompilerPluginComponent(
     new StdPhase(prev) {
       private val serializabilityTraitType = typeOf[SerializabilityTrait]
 
+      private val genericsNamesWithTypes = Map(
+        ("akka.actor.typed.Behavior", Seq(ClassType.Message)),
+        ("akka.actor.typed.ActorRef", Seq(ClassType.Message)),
+        ("akka.actor.typed.RecipientRef", Seq(ClassType.Message)),
+        ("akka.persistence.typed.scaladsl.ReplyEffect", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.projection.eventsourced.EventEnvelope", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.persistence.typed.scaladsl.Effect", Seq(ClassType.PersistentEvent)))
+
+      private val genericMethodsWithTypes = Map(
```

Comment:
While you're here, maybe also `pipeTo` and `forward` (at least these are the most commonly used ones)

Reply:
I just realised there is problem with `pipeTo` and `forward`. They are part of Akka Classic DSL. Signature of these methods:
```scala
def forward(message: Any)
def pipeTo(recipient: ActorRef)
```
I can only check part of `pipeTo`, as it's done by `final class PipeableFuture[T](val future: Future[T])` but no more.